dockers/docker-orchagent/docker-init.j2

```diff
@@ -58,3 +58,4 @@ if [ -f "$USE_PCI_ID_IN_CHASSIS_STATE_DB" ]; then
 fi
 
 exec /usr/local/bin/supervisord
+echo /usr/lib/libgcovpreload.so > /etc/ld.so.preload
```

Comment:
How about adding in the debian postinst step?

Reply:
How about adding in the debian postinst step?   ------updated
The package should be only for code coverage, not necessary to install it in product, right?  ----yes, this package is only for code coverage.